src/shared/components/palette/case-link/components/read-linked-cases.component.html

```diff
@@ -2,13 +2,31 @@
     <div class="govuk-!-margin-bottom-2">
         <h1 class="heading-h1">Linked cases</h1>
         <label class="table-sub-heading">This case is linked to</label>
-        <ccd-linked-cases-to-table [caseField]="caseField" (notifyAPIFailure)="getFailureNotification($event)"
+        <ccd-linked-cases-to-table [caseField]="caseField" (notifyAPIFailure)="getFailureLinkedToNotification($event)"
```

Comment:
It's not just to separate the error handling for from and to table rendering and it's more of tracking how many apis failed and at the same time we should able to see if all the api's failed or some of them, don't think its happening now ?